trytond/tests/test_tryton.py

```diff
@@ -1000,5 +1000,7 @@ def modules_suite(modules=None, doc=True):
         for test in test_mod.suite():
             if isinstance(test, doctest.DocTestCase) and not doc:
                 continue
+            if only and not any(x in str(test) for x in only.split(',')):
```

Comment:
faire le split en-dehors du for

Reply:
@JCavallo bien vu done (et renommage au passage)